Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -799,7 +799,7 @@ void CFrame::DoStop()
 
 			// Do not pause if netplay is running as CPU thread might be blocked
 			// waiting on inputs
-			bool should_pause = !NetPlayDialog::GetNetPlayClient();			
```

Comment:
this functions just returns a global variable. the best fix probably would be to make it not global, but in the interest of making the least breaking change, I just used the global and got rid of the weird get-a-global-pointer-by-reference function